portfolio/src/main/java/com/google/sps/data/UserLogin.java

```diff
@@ -0,0 +1,13 @@
+package com.google.sps.data;
+/**
+* User Login class to hold fields relating to the user login process
+*/
+public class UserLogin {
+  private final boolean status;
+  private final String redirectURL;
+
+  public UserLogin(boolean status, String redirectURL) {
+    this.status = status;
```

Comment:
nit: perhaps use a more descriptive name, e.g. isLoggedIn

Reply:
Got it! Also went through script.js to match the property names to these new variable names.